frontend/javascripts/libs/input.js

```diff
@@ -340,61 +339,32 @@ export class InputMouse {
 
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "mousedown",
-        this.targetSelector,
-        this.mouseDown,
-      ),
+      Utils.addEventListenerWithDelegation(document, "mousedown", targetSelector, this.mouseDown),
     );
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "mouseover",
-        this.targetSelector,
-        this.mouseOver,
-      ),
+      Utils.addEventListenerWithDelegation(document, "mouseover", targetSelector, this.mouseOver),
     );
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "mouseout",
-        this.targetSelector,
-        this.mouseOut,
-      ),
+      Utils.addEventListenerWithDelegation(document, "mouseout", targetSelector, this.mouseOut),
     );
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "touchstart",
-        this.targetSelector,
-        this.mouseOver,
-      ),
+      Utils.addEventListenerWithDelegation(document, "touchstart", targetSelector, this.mouseOver),
     );
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "touchend",
-        this.targetSelector,
-        this.mouseOut,
-      ),
+      Utils.addEventListenerWithDelegation(document, "touchend", targetSelector, this.mouseOut),
     );
     _.extend(
       this.delegatedEvents,
-      Utils.addEventListenerWithDelegation(
-        document,
-        "wheel",
-        this.targetSelector,
-        this.mouseWheel,
-        { passive: false },
-      ),
+      Utils.addEventListenerWithDelegation(document, "wheel", targetSelector, this.mouseWheel, {
+        passive: false,
+      }),
     );
 
-    this.hammerManager = new Hammer(this.domElement, {
+    this.hammerManager = new Hammer(domElement, {
```

Comment:
Could you add a comment referencing issue #3879 here, because I think this is still a problem for the touch handling. HammerJS binds to the specific element and doesn't support event delegation via a selector. Fixing this has low priority imo, so the comment should be sufficient :)